compiler/modules/nback/scaffold.bal

```diff
@@ -172,6 +174,8 @@ class Scaffold {
 
     function stackGuard() returns llvm:PointerValue => self.mod.stackGuard;
 
+    function getDebugLevel() returns DebugLevel => self.debugLevel;
```

Comment:
Instead of this, it's better to change the third argument to setDebugLocation to allow an additional value meaning set the debug location if the debugging level is full.

Reply:
fixed